prov/hostspool/hostspool_mgr_test.go

```diff
@@ -502,3 +520,252 @@ func testConsulManagerConcurrency(t *testing.T, cc *api.Client) {
 	err = cm.releaseWait("concurrent_host1", 500*time.Millisecond)
 	assert.Error(t, err, "Expecting concurrency lock for releaseWait()")
 }
+
+func testConsulManagerApply(t *testing.T, cc *api.Client) {
```

Comment:
Maybe this test can be split in order to improve the test excution time ?

Reply:
ok ! Will split the test into several tests organized by topic